lib/linguist/languages.yml

```diff
@@ -8330,3 +8330,12 @@ xBase:
   tm_scope: source.harbour
   ace_mode: text
   language_id: 421
+Vento:
+  type: markup
+  color: "#ff0080"
+  extensions:
+  - ".vto"
+  - ".vento"
```

Comment:
Samples and representative search links need to be provided for all extensions being added. Not, all extensions need to meet our usage requirements before inclusion so is one is less popular than another, it's probably best to remove it and people can rely on an [override](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md) to force the language until it meets the usage requirement for adding at a later date.

Reply:
Okay, I guess `.vto` is the right extension. Will remove `.vento`.